geo-traits/CHANGES.md

```diff
@@ -4,8 +4,8 @@
 
 - BREAKING: All traits now extend `GeometryTrait`.
   - <https://github.com/georust/geo/pull/1346>
-- Fix the lifetime annotation of `PointTrait` implemented for `geo_types::Point`.
```

Comment:
I don't think we need this changelog entry anymore since this pull request undoes those changes?